includes/DataObjects/UserRole.php

```diff
@@ -20,7 +20,7 @@ class UserRole extends DataObject
     /** @var string */
     private $role;
     /** @var int */
```

Comment:
Hmm, can this be removed now that there's a proper type annotation?

Reply:
Yes, done